src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    protected String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads task list from file.
+     *
+     * @return List of tasks loaded from file.
+     * @throws FileNotFoundException If file does not exist.
+     */
+    public List<Task> load() throws FileNotFoundException {
+        List<Task> tasks = new ArrayList<>();
+
+        // check if file exists
+        File f = new File(filePath);
+
+        if (f.exists()) {
+            Scanner s = new Scanner(f);
+            Task task;
+
+            while (s.hasNextLine()) {
+                String currTask = s.nextLine();
+                String[] taskSplit = currTask.split(" \\| ");
+
+                for (String c : taskSplit) {
+                    System.out.println(c);
+                }
+
+                if (taskSplit[0].equals("T")) {
+                    task = new Todo(taskSplit[2]);
+                } else if (taskSplit[0].equals("D")) {
```

Comment:
You can try implementing a guard clause for this `if else` cases, as there will never be a case where there will be 2 triggered cases.

It makes your code look more neater too! 
Here's a [link](https://deviq.com/design-patterns/guard-clause) for reference!

E.g 
```
if (a) {
    doA();
}

if (b) {
    doB();
}

if (c) {
    doC();
}
```